.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,5 @@
 * Link the PR to an issue for new features
 * Rebase your PR if it gets out of sync with master
+* Follow the contribution guidelines in `CONTRIBUTING.md`
```

Comment:
We should probably have a full link to `CONTRIBUTING.md`, i.e. https://github.com/grafana/grafana/blob/master/CONTRIBUTING.md.

Was thinking that including some sort of template like this could be useful:
`Fixes #<issue number>`

However, given the amount of text we have in our issue template and seems like 5% actually are using it it's maybe preferred to have as little information as possible and refer to the contributing doc?

Reply:
Full link for sure. 

Maybe fixes if it's short. I'll give it a try